Add permissions table for IDP

mesellings opened the pull request:
## Description

Add a permissions table for IDP features, as per Web Modeler roles.

Related to https://github.com/camunda/web-modeler/issues/13597.

<!-- Provide an overview of what to expect in the PR. -->
<!-- Relate or link the associated epic or task. -->
<!-- Add `@camunda/tech-writers` as reviewer to pull in a tech writer, or add your embedded tech writer. -->

## When should this change go live?

<!-- PRs merged go to stage.docs.camunda.io first and must be manually released to docs.camunda.io. -->

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**. (add `bug` or `support` label)
- [ ] This is already available but undocumented and should be released within a week. (add `available & undocumented` label)
- [ ] This is on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (create draft PR and/or add `hold` label)
- [X] This is part of a scheduled **alpha or minor**. (add alpha or minor label)
- [ ] There is **no urgency** with this change (add `low prio` label)

## PR Checklist

<!-- Camunda maintains 18 months of minor versions. Backporting your change to multiple versions is common. -->

- [X ] My changes are for **an upcoming minor release** and:
  - [X] are in the `/docs` directory (version 8.8).
  - [X] are in the `/versioned_docs/version-8.7/` directory (version 8.7).
- [ ] My changes are for an **already released minor** and are in a `/versioned_docs` directory.

<!-- UNCOMMENT THIS SECTION IF APPLICABLE. Adding or removing pages requires extra steps.
- [ ] I included my new page in the sidebar file(s).
- [ ] I added a redirect for a renamed or deleted page to the .htaccess file.
-->

- [X] I added a DRI, team, or delegate as a reviewer for technical accuracy and grammar/style:
  - [X] [Engineering team review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process)
  - [X] [Technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process) via `@camunda/tech-writers` unless working with an embedded writer.

<!-- UNCOMMENT THIS SECTION IF APPLICABLE. Changes to **docs infra**, including updates to workflows and adding new npm packages, must be first discussed via issue or #ask-c8-documentation and linked for context.
- [ ] My changes require a [docs infrastructure review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process). (add `dx` label) -->

sahilbhatoacamunda commented on versioned_docs/version-8.7/components/modeler/web-modeler/idp/idp-reference.md:
```diff
@@ -94,6 +94,30 @@ You can choose from the following supported LLM extraction models during [data e
+| View validate extraction                  |       Yes        |         Yes          |    Yes     |
+| Validate extraction (test documents)      |        No        |         Yes          |    Yes     |
+| Publish template                          |        No        |         Yes          |    Yes     |
+| View versions                             |       Yes        |          No          |    Yes     |
```
Same here as https://github.com/camunda/camunda-docs/pull/5428/files#r2023177168

mesellings commented on docs/components/modeler/web-modeler/idp/idp-reference.md:
```diff
@@ -94,6 +94,30 @@ You can choose from the following supported LLM extraction models during [data e
+| View validate extraction                  |       Yes        |         Yes          |    Yes     |
+| Validate extraction (test documents)      |        No        |         Yes          |    Yes     |
+| Publish template                          |        No        |         Yes          |    Yes     |
+| View versions                             |       Yes        |          No          |    Yes     |
```
Thanks - added

mesellings commented:
Thanks for the review @sahilbhatoacamunda - I've incorporated your feedback and updated the table - please re-review: 
![image](https://github.com/user-attachments/assets/c73c205e-4044-466e-93d4-7d3c5978cc9e)

christinaausley reviewed (approved):
This looks great!
